services/shhext/service.go

```diff
@@ -98,7 +98,14 @@ func (s *Service) InitProtocol(address string, password string) error {
 	if err := os.MkdirAll(filepath.Clean(s.dataDir), os.ModePerm); err != nil {
 		return err
 	}
-	persistence, err := chat.NewSQLLitePersistence(filepath.Join(s.dataDir, fmt.Sprintf("%x.db", address)), password)
+	oldPath := filepath.Join(s.dataDir, fmt.Sprintf("%x.db", address))
```

Comment:
Any reason for using `%x` instead of `%s`? The argument is a string.

Reply:
no, that was an initial mistake :), can't be corrected now as dev/nightly version have that db name, but we won't be using it anymore as the new path using `%s` and the installation-id

Reply:
We can add a comment 👍

Reply:
@cammellos how is the installationID set? does it come from status-react?

Reply:
I see it's in the NodeConfig, but just to understand how it is generated

Reply:
comes from `status-react` yes

Reply:
cool thank you!